lib/middleware.ts

```diff
@@ -81,11 +98,21 @@ var defaultHandlers = [
   // Response size
   (req, res) => ({"response-size": getResponseSize(res)}),
   // Response time (ns)
-  (req, res) => ({"response-time": getResponseTimeNs(req, res)}),
+  (req, res) => ({value: getResponseTimeNs(req, res)}),
   // Status code
   (req, res) => ({"status-code": res.statusCode}),
   // Ip address
   (req) => ({ip: getIp(req)}),
+
+  // Kayvee's reserved fields
+  // Log level
+  (req, res) => ({level: getLogLevel(req, res)}),
+  // Source
+  // -> Gets passed in among `options` during library initialization
+  // Title
+  () => ({title: "response-time"}),
+  // Type
+  () => ({type: "gauge"}),
```

Comment:
If flexible, I'll stick with this approach, since it makes these metrics more aligned with our generic metrics